### ISSUE TITLE
Tutorial: Semantic Search Documentation using deprecated elser service instead of elasticsearch service

### ISSUE THREAD

**Issue by eshaffer321** [elastic/elasticsearch#117829]
The “Tutorial: semantic search with semantic_text” in the Elasticsearch documentation currently uses the ELSER model, which has been deprecated as of version 8.16. However, the tutorial does not include any deprecation warning to notify users about this status. This can lead to confusion for users following the tutorial, as they may unknowingly rely on a deprecated service.

Steps to Reproduce:

	1.	Navigate to the tutorial: [Semantic Search Tutorial](https://github.com/elastic/elasticsearch/issues/link-to-tutorial-page).
	2.	Observe that there is no mention of ELSER being deprecated.
	3.	Compare with the [ELSER documentation](https://github.com/elastic/elasticsearch/issues/link-to-elser-page), which clearly states the deprecation notice:
	The ELSER service is deprecated as of version 8.16 and will be removed in a future release. Use the Elasticsearch inference service instead, with model_id included in service_settings.

Expected Behavior:

	•	The tutorial should clearly display a deprecation warning for the ELSER service.
	•	The warning should guide users to use the recommended Elasticsearch inference service with model_id in service_settings as an alternative.

Suggested Fix:
```
	 Deprecated in 8.16
The ELSER service used in this tutorial is deprecated and will be removed in a future release. Use the Elasticsearch inference service with model_id included in service_settings instead. Refer to the [ELSER documentation](https://github.com/elastic/elasticsearch/issues/link-to-elser-page) for details.
```

**Comment by eshaffer321** [comment]
https://github.com/elastic/elasticsearch/pull/117826

**Comment by davidkyle** [comment]
Thanks for raising this issue @eshaffer321 

The `elser` inference service was deprecated in 8.16 in favour of the `elasticsearch` inference service. The examples in the docs should be updated to use `"service": "elasticsearch"` instead of `"service": "elser"`. I agree this is confusing and the example API calls need to be updated.

cc @maxhniebergall

**Comment by maxhniebergall** [comment]
Thanks Dave. I agree we should update the tutorial. However, I think that rather than using a deprecation warning here, we should just change this tutorial to use the elasticsearch service. There is an example of this here: https://www.elastic.co/guide/en/elasticsearch/reference/current/infer-service-elasticsearch.html#inference-example-elasticsearch-elser

**Comment by davidkyle** [comment]
Fixed in #118007

**Comment by kosabogi** [comment]
> Fixed in [#118007](https://github.com/elastic/elasticsearch/pull/118007)

Thank you for working on this! I'll close this issue once #118007 is merged.
